worker.Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.22.0 AS build
+WORKDIR /go/src/subtrace
+COPY . .
+ARG SUBTRACE_CONTROL_URL
+RUN --mount=type=cache,target=/go/pkg --mount=type=cache,target=/root/.cache make
+
+FROM debian:12
+
+RUN apt-get update && apt-get install -y ca-certificates curl
+
+ARG TARGETARCH
+RUN cat | bash - <<EOF
+  curl -sLO "https://packages.clickhouse.com/deb/pool/main/c/clickhouse/clickhouse-common-static_23.9.6.20_${TARGETARCH}.deb"
+  curl -sLO "https://packages.clickhouse.com/deb/pool/main/c/clickhouse/clickhouse-server_23.9.6.20_${TARGETARCH}.deb"
+  DEBIAN_FRONTEND=noninteractive apt install -y ./clickhouse-common-static_23.9.6.20_${TARGETARCH}.deb ./clickhouse-server_23.9.6.20_${TARGETARCH}.deb
+  rm ./clickhouse-common-static_23.9.6.20_${TARGETARCH}.deb ./clickhouse-server_23.9.6.20_${TARGETARCH}.deb
+EOF
+
+COPY --from=build /go/src/subtrace/subtrace /usr/local/bin/subtrace
+
+RUN cat >/usr/local/bin/start_worker.sh <<EOF
+  CLICKHOUSE_DB=subtrace clickhouse-server --daemon
+  export SUBTRACE_CLICKHOUSE_HOST=localhost
+  export SUBTRACE_CLICKHOUSE_DATABASE=default
```

Comment:
Is there a reason we're using default instead of subtrace?

Reply:
I couldn't get clickhouse to automatically create a database called `subtrace`. It works in our compose.yaml setup, but for some reason I was struggling to get it working here.

(Btw I just noticed I forgot to remove the `CLICKHOUSE_DB=subtrace` from the server startup command, something leftover from my testing. Removed that part so that both clickhouse and the worker will run off the default database.)